dev/fattest.simplicity/src/componenttest/topology/utils/ExternalTestService.java

```diff
@@ -604,14 +604,29 @@ private static String getNetworkLocation() {
             return networkLocation;
         }
         String serverOrigin = System.getProperty(PROP_SERVER_ORIGIN);
-        if (!serverOrigin.startsWith("10.")) {
-            //It might be a 9.* address or a host name either way it is on the IBM9 Network
-            return "IBM9";
-        } else if (serverOrigin.startsWith("10.10")) {
-            return "HDCRTP";
-        } else if (serverOrigin.startsWith("10.34")) {
-            return "HDCHUR";
+	// Attempt to guess where the closest services will be located
+	if (serverOrigin.startsWith("9.20.")) {
+	    // Hursley
+	    return "IBM9UK";
+	} else if (serverOrigin.startsWith("9.42.") || serverOrigin.startsWith("9.46.")) {
+	    // RTP
+	    return "IBM9US";
+	} else if (serverOrigin.startsWith("9.30.")) {
+	    // SVL
+	    return "IBM9US";
+	}  else if (serverOrigin.startsWith("9.57.")) {
+	    // POK
+	    return "IBM9US";
+        } else if (serverOrigin.startsWith("10.34.") || serverOrigin.startsWith("10.36.")) {	    
+            return "HURPROD";
+	} else if (serverOrigin.startsWith("10.51.")) {
+	    return "FYREHUR";
+	} else if (serverOrigin.startsWith("10.17.") || serverOrigin.startsWith("10.11.") || serverOrigin.startsWith("10.15.")) {
+	    return "FYRESVL";
+	} else if (serverOrigin.startsWith("10.21.") || serverOrigin.startsWith("10.21.")) {
+	    return "FYRERTP";
```

Comment:
This line looks like it has a typo did you intend to repeat a check for `10.21.`?